src/Common/TrustedLogin/Trusted_Login_Config.php

```diff
@@ -0,0 +1,230 @@
+<?php
+/**
+ * TrustedLogin Configuration.
+ *
+ * Provides configuration building for TrustedLogin within the TEC plugin architecture,
+ * handling all constants and config array generation.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\TrustedLogin
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Common\TrustedLogin;
+
+use TEC\Common\Configuration\Configuration;
+use Tribe__Main;
+
+/**
+ * Configuration class for TrustedLogin functionality.
+ *
+ * This class is responsible for building the complete configuration array
+ * for TrustedLogin, including all constants and dynamic values.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\TrustedLogin
+ */
+class Trusted_Login_Config {
+
+	/**
+	 * Unique namespace for identifying the product in TrustedLogin.
+	 *
+	 * Used in hooks, filters, and URL generation to avoid conflicts
+	 * between multiple products using TrustedLogin.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const NAMESPACE = 'tec-common';
+
+	/**
+	 * URL for the vendor's support page.
+	 *
+	 * Displayed in the TrustedLogin UI and used as a fallback redirect if
+	 * the TrustedLogin service is unreachable.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const SUPPORT_URL = 'https://theeventscalendar.com/support/';
+
+	/**
+	 * Base URL where the TrustedLogin Connector plugin is installed.
+	 *
+	 * This URL is used by the TrustedLogin Client SDK to communicate with
+	 * the Connector plugin for granting and managing secure access. It must
+	 * match the site where the Connector plugin is active, such as
+	 * production or staging environments.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const WEBSITE_URL = 'https://my.theeventscalendar.com';
```

Comment:
"It must match the site where the Connector plugin is active, such as production or staging environments."

But it's a `const` - so it never changes from "https://my.theeventscalendar.com"?

Reply:
Either the code is wrong or the Docblock is.

Reply:
This looks like `vendor/website` which the TL docs define as:
> The URL to the vendor website. Must be a valid URL.

So **we** are the _vendor_ and the value seems correct.